### PR TITLE
NXBT-1450: Add simple nightwatch.js IT ftest package

### PR DESCRIPTION
**Pull request by vpasquier**

### CODE REVIEW

File: ftest/nightwatchjs/itests.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+  <property name="project.dir" value="${maven.basedir}"/>
+  <property name="out.dir" value="${maven.project.build.directory}"/>
+  <unzip dest="${out.dir}/" overwrite="false">
+    <artifact:resolveFile key="org.nuxeo:nuxeo-ftest::zip"/>
+  </unzip>
+  <import file="${out.dir}/nuxeo-ftest.xml"/>
+
+  <!-- Instruction for installing the MP to be tested and selenium to be run -->
+  <property name="mp.install" value="file:${out.dir}/thesource-marketplace-${maven.project.version}.zip"/>
```
Comment (troger) [inline ftest/nightwatchjs/itests.xml]: That's probably not the right name.

File: ftest/nightwatchjs/itests.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      <os family="windows"/>
+    </condition>
+    <exec executable="${cmd.npm}" failonerror="true">
+      <arg value="install"/>
+    </exec>
+    <exec executable="${cmd.npm}" spawn="true">
+      <arg value="start"/>
+    </exec>
+    <copy todir="${out.dir}">
+      <artifact:file key="org.nuxeo.ecm.distribution:nuxeo-marketplace-jsf-ui::zip" />
```
Comment (troger) [inline ftest/nightwatchjs/itests.xml]: The `nuxeo-marketplace-jsf-ui` is retrieved and copied here, but it's never installed. Default distribution in functional tests is now a Nuxeo Server.

File: ftest/nightwatchjs/itests.xml (hunk 1 above)
Comment (vpasquier) [inline ftest/nightwatchjs/itests.xml]: Oh thanks

File: ftest/nightwatchjs/itests.xml (hunk 2 above)
Comment (vpasquier) [inline ftest/nightwatchjs/itests.xml]: So I have missed some episodes. I don't handle the jsf mp anymore right?

**Pull request closed without merging**